src/app/shared/services/query.service.ts

```diff
@@ -11,10 +11,17 @@ import { UserActions } from '../interfaces';
 export class QueryService {
   readonly url = `${environment.baseUrl}/api/multieditor`;
   readonly schemaUrl = `${environment.baseUrl}/schemas/records`;
+  readonly matchTypesMap = {
```

Comment:
`private`

Reply:
more important thing is that this is kinda showing a some sort of bad design, I think it's better to unify this both user readable operation names and backend operation names.